### DISCUSSION TITLE
[NoDriver] Is there a generic way to wait for the page to be loaded?

### DISCUSSION THREAD

**Discussion by NareshMurthy** [ultrafunkamsterdam/undetected-chromedriver#2023] [Q&A]
Is there a way we can wait for the page to load completely, instead of relying on a element on the page `await tab.get(url)`? Like do we have something similar to `WaitForNetworkIdle0` or `DomContentLoaded` in puppeteer or playwright

**Comment by ultrafunkamsterdam** [comment]
```await tab.select('body') ```

**Reply by ultrafunkamsterdam** [reply]
As soon as the body is there, the page is loaded  and the function returns. After which you execute other code

**Reply by NareshMurthy** [reply]
Thanks @ultrafunkamsterdam
